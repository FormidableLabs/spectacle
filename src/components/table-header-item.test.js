import React from 'react';
import { mount } from 'enzyme';
import TableHeaderItem from './table-header-item';

describe('<TableHeaderItem />', () => {
  test('should render correctly', () => {
    const context = {
      styles: {
        components: {
          tableHeaderItem: {
            color: '#e01'
          }
        }
      }
    };
    const wrapper = mount(<TableHeaderItem>Header Text</TableHeaderItem>, {
      context
    });
    expect(wrapper).toMatchSnapshot();
  });
});
