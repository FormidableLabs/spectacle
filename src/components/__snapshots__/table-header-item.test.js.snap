// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TableHeaderItem /> should render correctly 1`] = `
<TableHeaderItem>
  <Styled(td)
    className={undefined}
    style={
      Array [
        Object {
          "color": "#e01",
        },
        Object {},
        Object {},
        undefined,
      ]
    }
  >
    <td
      className="css-0"
      style={
        Array [
          Object {
            "color": "#e01",
          },
          Object {},
          Object {},
          undefined,
        ]
      }
    >
      Header Text
    </td>
  </Styled(td)>
</TableHeaderItem>
`;
